### ISSUE TITLE
accessibility and screenreaders

### ISSUE THREAD

**Issue by denizsincar29** [daumling/ireal-renderer#8]
### i am a blind jazz pianist and i use a screenreader to use my computer. the thing is that your ireal renderer doesn't read the chords at all, it meens that there is no actual text to read out. so screenreaders cannot read images and they need text for that. i suggest to read about html aria elements. they are elements hidden to sighted users but red with the screenreader. so for example we can have image discryptions on websites that is read by the screenreader but not visible for sighted user.
for example you put an image on the website and then you put a text element or a hidden lable: (a dog is sitting on the floor and looking to small children playing in the other room.)
that text is not visible because you put an aria hidden attribute. but blind screenreader users can focus on that element, read it and select and copy it.

**Comment by denizsincar29** [comment]
sorry i am a bit mistaken. as i know, area hidden hides unnecessary for the blind users objects to avoid blind users to focus on them. for example images. we don''t need to focus on an image. but there is also some attribute maybe, or a css class, to hide a text from a normal user but do it focusable with screenreaders. there are many features for accessibility in html.
